### DISCUSSION TITLE
modify examples eyes

### DISCUSSION THREAD

**Discussion by yorch828** [Bodmer/TFT_eSPI#2880] [General]
Hello, I am a student and new to this platform, I would like to know how I could modify or add new styles to the eyes found in the examples in this library, since I would like to be able to change the color of the pupil or also add new designs, but how could I generate these changes and new designs?

**Comment by Bodmer** [comment]
This is not something I have done. There is a tutorial here that may hrlp:
https://learn.adafruit.com/animated-electronic-eyes/customizing
